PCbuild/_ctypes.vcxproj

```diff
@@ -61,7 +61,8 @@
   </PropertyGroup>
   <ItemDefinitionGroup>
     <ClCompile>
-      <AdditionalIncludeDirectories>..\Modules\_ctypes\libffi_msvc;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>..\Modules\_ctypes\libffi_msvc;$(libffiDir)include;$(libffiDir)src\x86;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```

Comment:
We still have `libffi_msvc` here, it should go away.

Reply:
done